src/controllers/signupController.js

```diff
@@ -0,0 +1,44 @@
+import express, { request } from "express"
+import mssql from 'mssql'
+import jwt from 'jsonwebtoken'
+import bcrypt from 'bcrypt'
+import dotenv from 'dotenv';
+const signupController=async(req,res)=>{
+    const salt=10;
+    const sql = "INSERT INTO users (userUUID, nume, prenume, email, parola) OUTPUT Inserted.userUuid VALUES (NEWID(), @nume, @prenume, @email, @parola)";
+  const sqlEmail = "SELECT email FROM users WHERE email = @email";
+
+  //verificare email-ului
+  const emailRequest = new mssql.Request();
+  emailRequest.input('email', mssql.VarChar, req.body.email);
```

Comment:
looks like we  need below a lot of details from req.body maybe destructre here and use it below